CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ The types of changes are:
 - `Security` in case of vulnerabilities.
 
 ## [Unreleased](https://github.com/ethyca/fides/compare/2.33.0...main)
+- Added erasure support for Iterable [#4695](https://github.com/ethyca/fides/pull/4695)
```

Comment:
```suggestion

### Added
- Added erasure support for Iterable [#4695](https://github.com/ethyca/fides/pull/4695)
```

Reply:
I differ to you! But just fyi Kelsey suggested I moved it to unreleased cause I missed getting it merged into this release. So let me know and I'll do whichever one is best =)